src/main/webapp/app/shared/sidebar/sidebar-card-item/sidebar-card-item.component.ts

```diff
@@ -9,4 +9,16 @@ import { SidebarCardElement, SidebarTypes } from 'app/types/sidebar';
 export class SidebarCardItemComponent {
     @Input() sidebarItem: SidebarCardElement;
     @Input() sidebarType?: SidebarTypes;
+    @Input() groupKey?: string;
+
+    removeChannelPrefix(name: string): string {
```

Comment:
Could you adapt the communication data before passing it to the sidebar-card.component?